src/find/matchers/group.rs

```diff
@@ -0,0 +1,141 @@
+use super::Matcher;
```

Comment:
Please add the license header

Reply:
I'm sorry I'm not sure which copyright holder to claim.
How about citing the project's License using the following text?

```
// This file is part of the uutils find package.
//
// For the full copyright and license information, please view the LICENSE
// file that was distributed with this source code.
```